.github/workflows/ci-cd.yml

```diff
@@ -356,6 +356,22 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Determine git commit sha and branch because does not provide them in a consistent manner
```

Comment:
Who doesn't provide them?

Reply:
Ahh... forgot to mention github actions. I updated the PR please check again.